sematic/resolvers/resource_managers/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Sematic
+from sematic.resolvers.resource_managers.in_memory_manager import (  # noqa: F401,E402
+    InMemoryResourceManager,
+)
+from sematic.resolvers.resource_managers.resource_manager import (  # noqa: F401,E402
+    ResourceManager,
```

Comment:
Why not have call sites import directly from their locations?
This encourages non granular Bazel dependencies, i.e. depending on `//sematic/resolvers:resource_managers` instead of `//sematic/resolvers/resource_managers:in_memory_manager`.

Reply:
done